charts/kueue/templates/manager/manager.yaml

```diff
@@ -23,6 +23,7 @@ spec:
       - args:
         - --config=/controller_manager_config.yaml
         - --zap-log-level=2
+        - --cert-dir=/tmp
```

Comment:
Yes, ```metrics-server``` also uses it:
- https://github.com/kubernetes-sigs/metrics-server/blob/9e899eaac0c1da92f0fe579f171d5251f44b1c08/manifests/base/deployment.yaml#L24

Reply:
What is this folder used for?

Reply:
This is the directory where the TLS certs are located. Without it, apiserver tries to create `apiserver.local.config/certificates` directory, but since it doesn't have sufficient permissions, it fails.